src/Tizen.CoreUI/src/CoreUI.UI/PositionManagerEntity.cs

```diff
@@ -0,0 +1,1041 @@
+    /// <since_tizen> 6 </since_tizen>
+    void ItemRemoved(int removed_index, CoreUI.Gfx.IEntity subobj);
+
+    /// <summary>The size of the items from <c>start_id</c> to <c>end_id</c> have been changed. The positioning and sizing of all items will be updated</summary>
+    /// <since_tizen> 6 </since_tizen>
+    /// <param name="start_id">The first item that has a new size</param>
+    /// <param name="end_id">The last item that has a new size</param>
+    void ItemSizeChanged(int start_id, int end_id);
+
+    /// <summary>The items from <c>start_id</c> to <c>end_id</c> now have their entities ready
+    /// 
+    /// The position manager will reapply the geometry to the elements if they are visible.</summary>
+    /// <since_tizen> 6 </since_tizen>
+    /// <param name="start_id">The first item that is available</param>
+    /// <param name="end_id">The last item that is available</param>
+    void EntitiesReady(uint start_id, uint end_id);
+
+    /// <summary>Translates the <c>current_id</c>, into a new id which is oriented in the <c>direction</c> of <c>current_id</c>. In case that there is no item, -1 is returned</summary>
+    /// <since_tizen> 6 </since_tizen>
+    /// <param name="current_id">The id where the direction is oriented at</param>
+    /// <param name="direction">The direction where the new id is</param>
+    /// <returns>The id of the item in that direction, or -1 if there is no item in that direction</returns>
+    int RelativeItem(uint current_id, CoreUI.UI.Focus.Direction direction);
+
+    /// <summary>Emitted when the aggregate size of all items has changed. This can be used to resize an enclosing Pan object.</summary>
+    /// <since_tizen> 6 </since_tizen>
+    /// <value><see cref="CoreUI.UI.PositionManager.EntityContentSizeChangedEventArgs"/></value>
+    event EventHandler<CoreUI.UI.PositionManager.EntityContentSizeChangedEventArgs> ContentSizeChangedEvent;
+    /// <summary>Emitted when the minimum size of all items has changed. The minimum size is the size that this position_manager needs to display a single item.</summary>
+    /// <since_tizen> 6 </since_tizen>
+    /// <value><see cref="CoreUI.UI.PositionManager.EntityContentMinSizeChangedEventArgs"/></value>
+    event EventHandler<CoreUI.UI.PositionManager.EntityContentMinSizeChangedEventArgs> ContentMinSizeChangedEvent;
+    /// <value><see cref="CoreUI.UI.PositionManager.EntityVisibleRangeChangedEventArgs"/></value>
+    event EventHandler<CoreUI.UI.PositionManager.EntityVisibleRangeChangedEventArgs> VisibleRangeChangedEvent;
+    /// <summary>This is the position and size of the viewport, where elements are displayed in. Entities outside this viewport will not be shown.</summary>
+    /// <since_tizen> 6 </since_tizen>
+    CoreUI.DataTypes.Rect Viewport {
```

Comment:
Do not make set-only property. 
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/property